test/ConfigDistributor.spec.js

```diff
@@ -86,55 +93,6 @@ describe('A DistributeConfig class', () => {
 
   });
 
-  describe('writeDirectory static method', () => {
-
-    context('when there is an error creating the directory', () => {
-
-      it('returns a promise that will be rejected', () => {
-        const invalidDirName = (function () {
-          const parts = new Array(256);
-          parts.fill('a');
-          return parts.join('');
-        }());
-        return expect(
-          ConfigDistributor.writeDirectory(invalidDirName)
-        ).to.be.rejected;
-      });
-
-    });
-
-    context('when there no error', () => {
-
-      let validDirName;
-
-      beforeEach(() => {
-        validDirName = path.join(__dirname, '/tmp/');
-        if (fs.existsSync(validDirName)) {
-          fs.rmdirSync(validDirName);
-        }
-        expect(fs.existsSync(validDirName)).to.be.false;
-      });
-
-      afterEach(() => {
-        fs.rmdirSync(validDirName);
-      });
-
-      it('returns a promise that will not be rejected', () => {
-        return expect(
-          ConfigDistributor.writeDirectory(validDirName)
-        ).to.not.be.rejected;
-      });
-
-      it('writes the directory', () => {
-        return ConfigDistributor.writeDirectory(validDirName).then(() => {
-          return expect(fs.existsSync(validDirName)).to.be.true;
-        });
-      });
-
-    });
-
-  });
-
   describe('writeFile static method', () => {
```

Comment:
`describe` that map perfectly onto methods/functions are often a smell:
https://testing.googleblog.com/2014/04/testing-on-toilet-test-behaviors-not.html
The exception is if the methods are completely decoupled and can be used independently, but at that point they don't have a reason to be in the same class.